bears/perl/PerlCriticBear.py

```diff
@@ -1,6 +1,10 @@
 from coalib.bearlib.abstractions.Linter import linter
 from dependency_management.requirements.DistributionRequirement import (
     DistributionRequirement)
+from dependency_management.requirements.AnyOneOfRequirements import (
+    AnyOneOfRequirements)
+from dependency_managment.requirements.PerlRequirement import (
```

Comment:
Is this dependency manager available in the [`coala package_manager`](https://gitlab.com/coala/package_manager/tree/master/dependency_management/requirements)?

Reply:
No it is not available, Thank you for pointing me in the right direction ... I will create `PerlRequirement` file.

Reply:
BTW the issue is not about creating a dependency. This issue is to create just Perl meta.

Reply:
I don't think there is a need to add `PerlRequirement` as `CpanRequirement` is already present..

Reply:
If `CpanRequirement` is the requirement here, then work with that. Adding `PerlRequirement` here without having it in package_manager will break it

Reply:
Will you be fixing this @Ishaan29 ?